R/stat-summary.r

```diff
@@ -270,13 +270,14 @@ mean_sdl <- wrap_hmisc("smean.sdl")
 #' @rdname hmisc
 median_hilow <- wrap_hmisc("smedian.hilow")
 
-#' Calculate mean and standard error
+#' Calculate mean and standard error of the mean
 #'
 #' For use with [stat_summary()]
 #'
-#' @param x numeric vector
-#' @param mult number of multiples of standard error
-#' @return A data frame with columns `y`, `ymin`, and `ymax`.
+#' @param x numeric vector.
+#' @param mult number of multiples of standard error.
+#' @return A data frame with columns `y` with the `mean`, `ymin` with the mean
+#' minus the standard error, and `ymax`, the mean plus the standard error.
```

Comment:
I think you can use definition lists here.

c.f. see "Lists" section of https://cran.r-project.org/web/packages/roxygen2/vignettes/rd-formatting.html

Btw, I think "minus/plus the standard error" should be something like "minus/plus the multiples of standard error"

Reply:
Changed!

Reply:
No, I meant definition lists. Please search for the text "Definition (named) lists" in the URL I pointed above. You cannot write it in Markdown, IIRC.